app/org/sagebionetworks/bridge/upload/TranscribeConsentHandler.java

```diff
@@ -39,9 +45,15 @@ public void handle(@Nonnull UploadValidationContext context) {
 
         // Also get external ID
         String userExternalId = options.get(ParticipantOption.EXTERNAL_IDENTIFIER);
+        
+        // And get user data groups
+        String dataGroupsSer = options.get(ParticipantOption.DATA_GROUPS);
+        Set<String> userDataGroups = (isNotBlank(dataGroupsSer))
+                ? Sets.newHashSet(Splitter.on(",").split(dataGroupsSer)) : null;
```

Comment:
Okay, added some more checks in HealthDataRecordTest and UploadArtifactsHandlerTest that values are being copied/transferred.